### DISCUSSION TITLE
Shapely.topology submodule

### DISCUSSION THREAD

**Discussion by fghasemi3** [HelgeGehring/gdshelpers#68] [General]
geometry\shapely_adapter.py imports shapely.topology
Recent shapely package does not seem to have a topology submodule. Other shapely submodules import normally. Should a different package be installed to support geometry\shapely_adapter.py in gdshelpers?

**Comment by f20221368-cyber** [comment]
even i am having the same problem. It says:ModuleNotFoundError: No module named 'shapely.topology'
